roxctl/helm/derivelocalvalues/extraction_test.go

```diff
@@ -0,0 +1,33 @@
+package derivelocalvalues
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_extractImageRegistry(t *testing.T) {
+	tests := map[string]struct {
+		imageName string
+		want      string
+		expectNil bool
+	}{
+		"":                             {"", "", true},
+		"registry/name:tag":            {"name", "registry", false},
+		"registry/repository/name:tag": {"name", "registry/repository", false},
+		// "name:tag":                     {"name", "", false},         // Won't work.
+		// "registry/name":                {"name", "registry", false}, // Won't work.
```

Comment:
Why it will not work?

Reply:
The regex in the implementation is too simple.